### PR TITLE
[TE] frontend - aaronucsd/Added application loading/error start

### PR DESCRIPTION
**Pull request by aaronucsd**
screen shot 2018-03-19 at 10 39 40 pm

1. Added new error route for catching other routes errors.
2. Added an application level loading handler

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { hash } from 'rsvp';
 import Route from '@ember/routing/route';
 import fetch from 'fetch';
+import RSVP from 'rsvp';
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js]: are we actually using rsvp here?

File: thirdeye/thirdeye-frontend/app/styles/pods/loading.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.route-loader-container {
+  position: absolute;
+  left: 48%;
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/styles/pods/loading.scss]: couldn't we achieve the same layout with flex box ? 
Personally not a big fan of magic numbers to position items. if you were trying to offset the spinner's own height and width. You could keep both `left` and `top` at 50%, and then use the transform() trick.
`transform: translateX(-50%) translateY(-50%);`

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js (hunk 1 above)
Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js]: Let me remove. I had it for testing a promise reject for the error. Thanks.

File: thirdeye/thirdeye-frontend/app/styles/pods/loading.scss (hunk 2 above)
Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/styles/pods/loading.scss]: i added the flexbox.

File: thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js (hunk 1 above)
Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js]: DONE

File: thirdeye/thirdeye-frontend/app/styles/pods/loading.scss (hunk 2 above)
Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/styles/pods/loading.scss]: DONE.

**Pull request merged**